src/libraries/System.Resources.Extensions/tests/BinaryFormatTests/Common/BasicObjectTests.cs

```diff
@@ -0,0 +1,83 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Linq;
+using System.Runtime.Serialization.Formatters;
+using BinaryFormatTests;
+using BinaryFormatTests.FormatterTests;
+
+namespace FormatTests.Common;
+
+public abstract class BasicObjectTests<T> : SerializationTest<T> where T : ISerializer
+{
+    private protected abstract bool SkipOffsetArrays { get; }
+
+    [Theory]
+    [MemberData(nameof(SerializableObjects))]
+    public void DeserializeStoredObjects(object value, TypeSerializableValue[] serializedData)
+    {
+        _ = value;
+
+        int platformIndex = serializedData.GetPlatformIndex();
+        for (int i = 0; i < serializedData.Length; i++)
+            for (FormatterAssemblyStyle assemblyMatching = 0; assemblyMatching <= FormatterAssemblyStyle.Full; assemblyMatching++)
```

Comment:
While unbraced `if` is allowed in the top method scope, the rule for omitting braces is for a single line, not a single statement.

So the for loop on line 22 should use braces, as it contains lines 23-39.  39-23+1 = 17, 17 > 1.

Reply:
Another comment marked as resolved but without being addressed or rebutted... is a commit missing?